src/Chainweb/Pact/PactService.hs

```diff
@@ -1,5 +1,5 @@
 {-# LANGUAGE BangPatterns #-}
-{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE DerivingStrategies #-}
```

Comment:
I just checked and `DeriveGeneric` wasn't needed so I assume you can remove it instead of changing it.